### PR TITLE
Dread: Speed Booster In Dairon Freezer

### PR DESCRIPTION
**Pull request by Tyranisaur**

### CODE REVIEW

**Comment by ThanatosGit** [comment]
Is it possible to get out there with morph + (cross) bombs? Never tried it but thought I just ask.

**Comment by Tyranisaur** [comment]
> Is it possible to get out there with morph + (cross) bombs? Never tried it but thought I just ask.

So it's possible with cross bombs both with a horizontal one paired with a wall jump, and a vertical one paired with the water bomb jump usage.

I imagine that it's probably possible with normal bombs as well, but it's either too hard for me, or takes too long for me to to pull off right off the bat.

**Comment by ThanatosGit** [comment]
I think that's not in logic Could you add that to this pr?
https://github.com/randovania/randovania/assets/117127188/a387d5c8-ba97-4412-ba1a-7503619af86d

File: CHANGELOG.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added: Allow using Shinesparks in Gravity Suit Tower by storing speed in the upper part of Gravity Suit Room, also when Door Lock rando is enabled.
 - Added: Pseudo-Wave Beam to break the blob in Ferenia Wave Beam Tutorial, from the right.
 - Added: Use Spider Magnet with Grapple Beam in Ghavoran Spider Magnet Elevator.
+- Added: Use Speed Booster to get past the pool of water in Dairon Freezer before turning on the power.
+- Added: Various trick alternatives to get paset the pool of water in Dairon Freezer with Bomb Jumps.
```
Comment (ThanatosGit) [inline CHANGELOG.md:69]: typo `paset`

**Review by ThanatosGit — CHANGES_REQUESTED** [review]

**Review by ThanatosGit — APPROVED** [review]

**Pull request merged**